tbtc-ts/test/data/bitcoin.ts

```diff
@@ -43,3 +45,230 @@ export const testnetUTXO: UnspentTransactionOutput & RawTransaction = {
+        blindingFactor: BigNumber.from("0xf9f0c90d00039523"),
+        createdAt: 1641650400,
+      },
+    },
+    {
+      // P2WSH deposit
+      hash: "5c54ecdf946382fab2236f78423ddc22a757776fb8492671c588667b737e55dc",
+      rawTx: {
+        transactionHex:
+          "01000000000101a0367a0790e3dfc199df34ca9ce5c35591510b6525d2d586916672" +
+          "8a5ed554be0100000000ffffffff02e02e00000000000022002086a303cdd2e2eab1" +
+          "d1679f1a813835dc5a1b65321077cdccaf08f98cbf04ca962c2c1100000000001600" +
+          "14e257eccafbc07c381642ce6e7e55120fb077fbed0247304402206dafd502aac9d4" +
+          "d542416664063533b1fed1d16877f0295740e1b09ec2abe05102200be28d9dd76863" +
+          "796addef4b9595aad23b2e9363ac2d64f75c21beb0e2ade5df0121039d61d62dcd04" +
+          "8d3f8550d22eb90b4af908db60231d117aeede04e7bc11907bfa00000000",
+      },
+      utxo: {
+        transactionHash:
+          "5c54ecdf946382fab2236f78423ddc22a757776fb8492671c588667b737e55dc",
+        outputIndex: 0,
+        value: 12000,
+      },
+      depositData: {
+        ethereumAddress: "0x934B98637cA318a4D6E7CA6ffd1690b8e77df637",
+        amount: BigNumber.from(12000),
+        refundPublicKey:
+          "039d61d62dcd048d3f8550d22eb90b4af908db60231d117aeede04e7bc11907bfa",
+        blindingFactor: BigNumber.from("0xf9f0c90d00039523"),
+        createdAt: 1641650400,
+      },
+    },
+  ],
+  previousSweepUtxo: NO_PREVIOUS_SWEEP,
+  sweepResult: {
+    transactionHash:
+      "f8eaf242a55ea15e602f9f990e33f67f99dfbe25d1802bbde63cc1caabf99668",
+    transaction: {
+      transactionHex:
+        "01000000000102bc187be612bc3db8cfcdec56b75e9bc0262ab6eacfe27cc1a699" +
+        "bacd53e3d07400000000c948304502210089a89aaf3fec97ac9ffa91cdff59829f" +
+        "0cb3ef852a468153e2c0e2b473466d2e022072902bb923ef016ac52e941ced78f8" +
+        "16bf27991c2b73211e227db27ec200bc0a012103989d253b17a6a0f41838b84ff0" +
+        "d20e8898f9d7b1a98f2564da4cc29dcf8581d94c5c14934b98637ca318a4d6e7ca" +
+        "6ffd1690b8e77df6377508f9f0c90d000395237576a9148db50eb52063ea9d98b3" +
+        "eac91489a90f738986f68763ac6776a914e257eccafbc07c381642ce6e7e55120f" +
+        "b077fbed8804e0250162b175ac68ffffffffdc557e737b6688c5712649b86f7757" +
+        "a722dc3d42786f23b2fa826394dfec545c0000000000ffffffff01488a00000000" +
+        "00001600148db50eb52063ea9d98b3eac91489a90f738986f60003473044022037" +
+        "47f5ee31334b11ebac6a2a156b1584605de8d91a654cd703f9c843863499740220" +
+        "2059d680211776f93c25636266b02e059ed9fcc6209f7d3d9926c49a0d8750ed01" +
+        "2103989d253b17a6a0f41838b84ff0d20e8898f9d7b1a98f2564da4cc29dcf8581" +
+        "d95c14934b98637ca318a4d6e7ca6ffd1690b8e77df6377508f9f0c90d00039523" +
+        "7576a9148db50eb52063ea9d98b3eac91489a90f738986f68763ac6776a914e257" +
+        "eccafbc07c381642ce6e7e55120fb077fbed8804e0250162b175ac6800000000",
+    },
+  },
+}
+
+export const sweepWithPreviousSweep: SweepDepositsTestData = {
```

Comment:
`sweepWithMainUtxo`?

Reply:
Done in https://github.com/keep-network/tbtc-v2/pull/96/commits/4dccc33c9feb9252e2577f323da7294a691b3569.